.github/workflows/rust.yml

```diff
@@ -28,11 +28,11 @@ jobs:
         with:
           command: build
           args: --workspace --all-features
-      - name: Build (no_std)
+      - name: Build wasmi itself as no_std
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --workspace --no-default-features
+          args: --lib --no-default-features --target thumbv7em-none-eabi
```

Comment:
We cannot use `--lib` here because this would only build the legacy `wasmi` engine but not the newer `wasmi_v1`.

Reply:
`--workspace --exclude wasmi_cli` should work instead

Reply:
However, it seems that the `thumbv7em-none-eabi` is not only missing the `std` library but also even the `alloc` and `core` libraries. I just ran this locally and saw some weird error messages such as:
```
error[E0405]: cannot find trait `Copy` in this scope
   --> /home/.cargo/registry/src/github.com-1ecc6299db9ec823/num-traits-0.2.15/src/lib.rs:617:31
    |
617 |     fn compute<T: NumAssign + Copy>(mut x: T, y: T) -> T {
    |                               ^^^^ not found in this scope
```

Reply:
It'll still need some limiting to libraries as the src/bin/instantiate.rs is clearly std dependent. As for the alloc and core, that I can probably add via the toolchain action.

Reply:
Looks all good now as far as I can tell, please clear for CI run again. (Adding two targets is a bit weird because the actions docs don't say it's an array, but it seems to work that way with other properties of the action, so meh, probably it works).